projects/core/src/user/events/user.events.ts

```diff
@@ -3,22 +3,17 @@ import { Address } from '../../model/address.model';
 
 export abstract class UserAddressEvent extends CxEvent {}
 
-export class UserAddressUpdateEvent extends UserAddressEvent {
-  static readonly type = 'UserAddressUpdateEvent';
+export class UpdateUserAddressEvent extends UserAddressEvent {
+  static readonly type = 'UpdateUserAddressEvent';
   addressId: string;
```

Comment:
can we just pass `addressId`?

Reply:
I don’t think we have control over the payload.  From the tests I did so far, the action payload is copied to the event when using the event builder pattern